server/plugin.ts

```diff
@@ -240,12 +135,10 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     // Register server side APIs
     defineRoutes(router);
     defineAuthTypeRoutes(router, config);
+
     // set up multi-tenent routes
-    if (config.multitenancy?.enabled) {
+    if (config.multitenancy.enabled) {
```

Comment:
Why remove the ternary check here?

Reply:
probably missed that during development or merge, rebase